ui/app/src/App.tsx

```diff
@@ -246,6 +246,18 @@ function App() {
                 <p> Creator: {selectedStream.creator}</p>
                 <p> Amount per second: {String(streamInfo.amountPerSecond)}</p>
                 <p> Funded amount: {String(streamInfo.fundedAmount)}</p>
+                <p>
+                  <Link
+                    to={`/${selectedStream.creator}/${selectedStream.streamId}`}
```

Comment:
Does this need stream manager address too?

Reply:
lol this PR is not ready at all that's why I left it as draft

Reply:
still plenty to be done before it's ready for review!